files/class-image.php

```diff
@@ -157,6 +162,18 @@ public function get_mime_type() {
 		return $this->mime_type;
 	}
 
+	/**
```

Comment:
would be good to note the file size measuring unit  whether it be bytes, kb, or mb just so its clear what this integer represents in case someone ends up in this code later.